packages/server/src/profile/profile.controller.ts

```diff
@@ -323,8 +323,6 @@ export class ProfileController {
 
     const pendingCourses = await this.profileService.getPendingCourses(user.id);
 
-    const organizationRole =
```

Comment:
Do we need to remove this?

Reply:
organizationRole is in the organization object, it is not needed here.